### PR TITLE
feat(query): allow url to be reactive

### LINKED ISSUE #2: feat: manage dynamics params in url
Ex: `PUT /users/:id`

### PR DESCRIPTION
**Pull request by gjuchault**
Closes #2

### CODE REVIEW

**Comment by lonk** [comment]
draft: need to test it + deploy on useMutation

**Comment by lonk** [comment]
Good for me, works properly

File: src/query.ts [hunk 1]
```diff
@@ -59,10 +59,7 @@ export function useQuery<
     });
   }
 
-  watch(
-    () => url.value,
-    (_, __, onInvalidate) => makeQuery(onInvalidate)
-  );
+  watch(url, (_, __, onInvalidate) => makeQuery(onInvalidate));
```
Comment (punkeel) [inline src/query.ts:62]: That's a 10x improvement if you ask me

Comment (PymZoR) [reply src/query.ts:62]: I ask you

**Pull request merged**